src/webgl/p5.Shader.js

```diff
@@ -1379,6 +1358,7 @@ p5.Shader = class Shader {
     return this.uniforms.uStrokeWeight !== undefined;
```

Comment:
strokeShader was not working in the docs. This was because of two reasons:

1. As I mentioned you need to remove this method created here https://github.com/processing/p5.js/pull/7256#issuecomment-2345767755
2. 2nd reason is since you are using the same name for strokeShader as we have in shaderHooks, it will clash. We will manage it afterwords but for testing purpose it would be nice if you change your method name from strokeShader to something else.